content/applications/marketing/events/event_templates.rst

```diff
@@ -0,0 +1,219 @@
+===============
+Event templates
+===============
+
+The Odoo *Events* application provides the ability to customize and configure event templates, which
+can be used to expedite the event-creation process.
+
+These templates can be created and personalized in the application, and then selected from an event
+form, in order to quickly apply a series of settings and elements to the new event, all of which can
+be further modified, if needed.
+
+Event templates page
+====================
+
+In the Odoo *Events* application, event templates can quickly be created and modified.
+
+To begin, navigate to :menuselection:`Events app --> Configuration --> Event Templates`. Doing so
+reveals the :guilabel:`Event Templates` page. Here, find all the existing event templates in the
+database.
+
+.. image:: event_templates/event-templates-page.png
+   :align: center
+   :alt: The Event Templates page in the Odoo Events application.
+
+By default, Odoo provides three pre-configured event templates: :guilabel:`Exhibition`,
+:guilabel:`Training`, and :guilabel:`Sport`, which all have their own unique customizations applied
+to them.
+
+To change how these event templates appear on the *Template* drop-down field on an event form,
+drag-and-drop them into any desired order, using the :guilabel:`six block` icons, located to the
+left of each event template line on the :guilabel:`Event Templates` page.
```

Comment:
You can use the `oi-draggable` icon (_which is coming soon in a PR near you_ https://github.com/odoo/documentation/pull/9256), it will appear once the PR is merged :)
```suggestion
drag-and-drop them into any desired order, using the :icon:`oi-draggable` :guilabel:`(draggable)`
icon, located to the left of each event template line on the :guilabel:`Event Templates` page.
```